### PR TITLE
Fix output option

### PR DESCRIPTION
**Pull request by SteveALee**
I wanted to use this and found it broken.
Note, the output path is a relative path to root or cwd. eg ```funcpack pack .\functionApp\src -o ..\build```

### CODE REVIEW

**Comment by SteveALee** [comment]
Well I visited the web form and am now waiting for the email to sign.

**Comment by SteveALee** [comment]
Very smooth CLA process :)

File: src/utils/index.ts [hunk 1]
```diff
@@ -1,2 +1,2 @@
 export * from "./fs-helper";
-export * from "./config-loader";
```
Comment (SteveALee) [inline src/utils/index.ts]: As as I set code to give CRLF this is probably as project is relying  on global git config for EOL handling Should be a project setting.

File: src/packhost-generator.ts [hunk 2]
```diff
@@ -85,7 +85,7 @@ export class PackhostGenerator {
                     directory: dir,
                 });
                 return null;
-                //throw new Error(`Function ${name} does not have a valid start file`);
+                // throw new Error(`Function ${name} does not have a valid start file`);
```
Comment (christopheranderson) [inline src/packhost-generator.ts]: formatting - should be fine

**Comment by christopheranderson** [comment]
Two issues:
 - I also wanted you to not have to use ".funcpack" as the name, so we should let you override that: <issue TBC>
- Breaks in strange ways when you don't provide a directory for output that that already exists: <issue TBC>

Otherwise, PR looks good, tests good. I'll pull this into the next release.

**Pull request merged**